src/bot.py

```diff
@@ -138,7 +137,7 @@ async def run_tasks(self, contributor: Contributor) -> None:
             logging.warning("No contributor for the given time period.")
 
     @staticmethod
-    @aiocron.crontab("0 0 1 * *")
+    @aiocron.crontab(f"0 0 {bot_settings.time_period_days} * *")
```

Comment:
Does the PR fix the logic for this as well?

Reply:
I've suggested the alternate change above about this.
Using the same variable for different purposes seems not that good imho.

Reply:
![image](https://user-images.githubusercontent.com/60746512/198639946-e6711704-b7df-44e8-b908-3c9becdeb734.png)

Reply:
> ![image](https://user-images.githubusercontent.com/60746512/198639946-e6711704-b7df-44e8-b908-3c9becdeb734.png)

We are not posting on a specific day of the month .. instead every n day.. for example, if it is 7 .. 00:00 currently and +7 days and so on

Reply:
@diwash007 Seems like a good bet since we'd be running the bot at a precise date (rendering Contributor of the Week to Contributor of the Month). Thoughts?

Reply:
Okay got it!